hack/lib/Dockerfile-origin-release

```diff
@@ -0,0 +1,9 @@
+FROM registry.svc.ci.openshift.org/openshift/origin-release:v4.0 as origin-release
```

Comment:
I don't know jq to assess this. Probably does what inplace sed could do easier.

Reply:
if you write me the multiline, multi-replace sed command, sure. Otherwise this jq does the job.